ls1intum/Artemis
# [Enhancement] Let instrucotrs set student exams to submitted/unsubmitted

### Checklist

### Motivation and Context


With this change instructors can set the` submitted` state of individual student exams. E.g.: if a student did not manage to submit the exam in time and the instructor decides the exam should count nonetheless, he can set the exam to submitted by himself.

### Description

In the student-exam-detail following things were changed:
1. Instructors can see the Matriculation number now.
2. I added a "Change Student Exam to Submitted" button, where the submission status can be  toggled.
<img src="https://user-images.githubusercontent.com/33342534/109276945-f0d47080-7816-11eb-9ac1-455a48abd2c6.png" width="350">
If the exam was not submitted:
<img src="https://user-images.githubusercontent.com/33342534/109277191-46108200-7817-11eb-9944-a4f5acacb5bc.png" width="350">
If it was submitted, but the instructor wants to set it to "unsubmitted":
<img src="https://user-images.githubusercontent.com/33342534/109277465-ab647300-7817-11eb-931d-aabcc3b124cb.png" width="350">

### Steps for Testing


1. create an exam and start the participation with a student. Don't hand  in the exam.
2. After the exam is over go to exam -> student-exams -> View (of the exam which you did start, but not hand in)
3. Use the "Change Student Exam to Submitted" button.
4. Go to the student account and check if the now handed in exam is visible.
<img src="https://user-images.githubusercontent.com/33342534/109278174-84f30780-7818-11eb-856b-2c9233620e76.png" width="350">
--->
<img src="https://user-images.githubusercontent.com/33342534/109278266-a0f6a900-7818-11eb-8574-8ed3903dc798.png" width="350">
